roles/servicetelemetry/templates/manifest_snmp_traps.j2

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: snmp-webhook
+spec:
+  selector:
+    matchLabels:
+      app: snmp-webhook
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: snmp-webhook
+    spec:
+      containers:
+        - name: prometheus-webhook-snmp
+          image: 'quay.io/mrunge/prometheus-webhook-snmp:latest'
+          ports:
+            - containerPort: 9099
+          env:
+            - name: SNMP_COMMUNITY
+              value: public
+            - name: SNMP_RETRIES
+              value: "1"
+            - name: SNMP_HOST
+              value: "{{ snmp_trap_host }}"
```

Comment:
Can you specify a host of 0.0.0.0 or something to listen on all available interfaces?

Reply:
Disregard